### ISSUE TITLE
Feature: Ability to include standard text at the top of all files

### ISSUE THREAD

**Issue by mrlacey** [Microsoft/WindowsTemplateStudio#477]
It's common to want a specific piece of text at the top of each file (of certain file types) often as a copyright header and it would be nice if this could be added by just adding a feature during project generation.

Some questions:
- How best to specify what the contents of the header should be in the wizard?
- Which file types to add this to? Just *.cs, or *.xaml as well?

**Comment by ralarcon** [comment]
Initially I was thinking the scope of this limited to the code that we generate including our license disclaimer but, if i'm not wrong, this is something you want to include as an end-user wts feature in the wizard...   

We can start using this feature as a paradigm and discovery for features which requires extra user inputs. Good chance to start this area.

**Comment by mrlacey** [comment]
Also related to the idea of "enhanced pages" #325

**Comment by crutkas** [comment]
idea here is a user wants this out of the gate for their solution?  clever.

**Comment by mrlacey** [comment]
Yes, the idea here is that if I'm using this to build an app I want copyright notice and comment headers in all files. I want these included in the generated project, not to have to go and add to all the files that are created.

**Comment by crutkas** [comment]
Great idea. Jelly I didn't think of it :)
